WORKSPACE

```diff
@@ -44,18 +44,25 @@ http_archive(
 
 # Kotlin
 
-RULES_KOTLIN_VERSION = "legacy-modded-1_0_0-01"
-
-RULES_KOTLIN_SHA = "b7984b28e0a1e010e225a3ecdf0f49588b7b9365640af783bd01256585cbb3ae"
-
-http_archive(
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+git_repository(
     name = "io_bazel_rules_kotlin",
-    sha256 = RULES_KOTLIN_SHA,
-    strip_prefix = "rules_kotlin-%s" % RULES_KOTLIN_VERSION,
-    type = "zip",
-    urls = ["https://github.com/cgruber/rules_kotlin/archive/%s.zip" % RULES_KOTLIN_VERSION],
+    remote = "https://github.com/cromwellian/rules_kotlin.git",
+    commit = "353329496db7817fa99d18ebec3c76410bdbc6a0"
 )
 
+#RULES_KOTLIN_VERSION = "legacy-modded-1_0_0-01"
+
+#RULES_KOTLIN_SHA = "b7984b28e0a1e010e225a3ecdf0f49588b7b9365640af783bd01256585cbb3ae"
+
+#http_archive(
+#    name = "io_bazel_rules_kotlin",
+#    sha256 = RULES_KOTLIN_SHA,
+#    strip_prefix = "rules_kotlin-%s" % RULES_KOTLIN_VERSION,
+#    type = "zip",
+#    urls = ["https://github.com/cgruber/rules_kotlin/archive/%s.zip" % RULES_KOTLIN_VERSION],
+#)
+
```

Comment:
I'd prefer you just deleted this. It'll always be in the git history for when we want to revert it.